src/main/java/edu/ksu/canvas/attendance/form/RosterFormValidator.java

```diff
@@ -12,6 +13,8 @@
 @Component
 public class RosterFormValidator implements Validator {
 
+    private static final Logger LOG = Logger.getLogger(RosterFormValidator.class);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "LOG" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1068)